src/main/scala/tv/teads/github/api/util/HttpClient.scala

```diff
@@ -72,6 +73,23 @@ case class HttpClient private (uri: Uri,
         FailedRequest(response.status)
     }
 
+  def postJson[T](entity: T)(implicit w: Marshaller[T]): Future[HttpResponse] = {
+    val pipeline = sendReceive
+    pipeline(Post(uri, entity))
+  }
+
+  def postJsonInto[T, O](entity: T)(implicit w: Marshaller[T], evidence: FromResponseUnmarshaller[O]) = {
```

Comment:
Why do you basically copy the content of the `executeRequest` method ? 
I think that HttpClient can be reworked to handle POSTs wtihout introducing duplication, eg. by passing a `spray.httpx.RequestBuilder` instead of an Uri.